core/src/main/java/com/linecorp/armeria/internal/client/ClientUtil.java

```diff
@@ -179,6 +179,9 @@ O executeWithFallback(U execution,
         try {
             return execution.execute(ctx, req);
```

Comment:
What do you think about moving the call to `completeLogIfIncomplete` from line 111 to here? It seems like it would prevent the case where an http response is returned in the pre decorator.

Reply:
Shouldn't we also change this line?
`futureConverter, errorResponseFactory, req, false);`
https://github.com/line/armeria/blob/89e1a840079258251a656c9cf7807952fa28a32c/core/src/main/java/com/linecorp/armeria/internal/client/TailPreClient.java#L82

Additionally, this isn't related to this PR but shouldn't `tryCompleteLog` be `true` in these lines? cc @ikhoon 
https://github.com/line/armeria/blob/51f07909ffcf897fb782531048a77bfc485c3cd7/core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java#L325-L328

Reply:
@ikhoon Do you have any opinion on this?

Reply:
Side note) I couldn’t immediately tell the difference between the two just from the function name `executeWithFallback`. Renaming them to something more explicit like `executePreClient` and `executeClient` would make the context clearer during code review.